### DISCUSSION TITLE
Make the same category display the same color on different articles.

### DISCUSSION THREAD

**Discussion by MCSeekeri** [CaiJimmy/hugo-theme-stack#887] [Ideas]
Although the way it's handled now is fine, I would like to make the same category display the same color on different articles, so that when I have more articles, visitors can find the article they want to read more quickly according to the color.
(English is not my first language and I use a translator)

**Comment by Lovirr** [comment]
标签颜色可以自定义的

**Reply by MCSeekeri** [reply]
可否稍微详细的说一下怎么自定义，因为文档似乎404了，而我目前只弄明白怎么让所有的分类都显示一个颜色…………

**Reply by CaiJimmy** [reply]
@MCSeekeri 抱歉文档服务器应该是没配置好所以 404 了，我刚刚才发现。

可以参考这个模板来给分类添加颜色：https://github.com/CaiJimmy/hugo-theme-stack-starter/blob/master/content/categories/example-category/_index.md

**Reply by MCSeekeri** [reply]
了解了，感谢。
